flexget/plugins/input/anidb_list.py

```diff
@@ -26,69 +28,100 @@ class AnidbList(object):
                 'error_pattern': 'user_id must be in the form XXXXXXX'},
             'type': {
                 'type': 'string',
-                'enum': ['shows', 'movies'],
+                'enum': ['shows', 'movies', 'ovas'],
                 'default': 'movies'},
+            'mode': {
+                'type': 'string',
+                'enum': ['all', 'undefined', 'watch', 'get', 'blacklist', 'buddy'],
+                'default': 'all'},
+            'pass': {
+                'type': 'string'},
             'strip_dates': {
                 'type': 'boolean',
-                'default': False}
+                'default': False},
+            'user_agent': {
```

Comment:
We already have a plugin for this though: https://flexget.com/Plugins/headers

Reply:
I'm not sure on my implementation of the headers in the plugin. I now check if the headers and user-agent were already set. If they were, those are used. Otherwise it will fall back to the plugin's default user-agent.